tox.ini

```diff
@@ -1,17 +1,30 @@
 [tox]
 minversion = 3.2.0
-envlist=py,linter,mypy
+envlist=py,linter,mypy,coverage-report
 
 [testenv]
 extras = test
+set_env =
+    COVERAGE_PROCESS_START={toxinidir}/pyproject.toml
+commands_pre =
+    py,py3{11,12},e2e: {envpython} -c 'import pathlib; pathlib.Path("{env_site_packages_dir}/cov.pth").write_text("import coverage; coverage.process_startup()")'
```

Comment:
Maybe we only want this in the `e2e` settings? Otherwise we'll get it for docs, the cli env, etc.

Reply:
It's configured for `py`, `py311, `py312` and `e2e`. I could limit it to `e2e`.

Reply:
I have limited the external coverage tracking to `e2e`.

Reply:
Ah, I didn't recognize that prefix syntax.